db/models/model.js

```diff
@@ -53,8 +53,25 @@ function fetchArticleFromDatabase(articleId) {
   });
 }
 
+function fetchCommentsFromDatabase(articleId) {
+  return new Promise((resolve, reject) => {
```

Comment:
PSQL is already returning a resolved or rejected promise here no need to create a new instance of the promise class